### DISCUSSION TITLE
How can i flag the -lwiringPi ??

### DISCUSSION THREAD

**Discussion by ghost** [nasa/cFS#435] [Q&A]

**Comment by jphickey** [comment]
I assume this is a link library you are referring to?

Simple option (kind of a hack but will get you there):  set the variable `OSAL_LINK_LIBS` in your toolchain file.  It is empty by default.  This list is just put into the `target_link_libraries()` for CFE core.

Better option: If you have a custom PSP layer for your board, then add it to "target_link_libraries" for your PSP (because PSP is statically linked with CFE core, this also gets translated to final link)

Third option: You can override the `cfe_exec_do_install` function in your `arch_build_custom.cmake` file to provide any other sort of directives/customization of the final target.  (Caveat: this works in theory but not really tested).
